src/writer/journal.py

```diff
@@ -43,6 +43,7 @@ def __init__(
         else:
             self.trigger["component"]["type"] = "block"
             component = graph.get_start_nodes()[0].component
+            self.trigger["component"]["id"] = component.id
```

Comment:
@VladKrasny for the API calls and Cron, we need to add the id as well, otherwise FE is crashing while listing blocks.